addons/attachment_indexation/models/ir_attachment.py

```diff
@@ -140,3 +140,8 @@ def _index(self, bin_data, mimetype, checksum=None):
         if checksum:
             index_content_cache[checksum] = res
         return res
+
+    def copy(self, default=None):
+        if self.index_content:
+            index_content_cache[self.checksum] = self.index_content
+        return super(IrAttachment, self).copy(default=default)
```

Comment:
That's ancestral-style Python
```suggestion
        return super().copy(default=default)
```